tests/images.rs

```diff
@@ -17,10 +17,13 @@ use testcontainers::{core::WaitFor, *};
 const RANDOM_PORTS: Range<u16> = 32768..61000;
 
 #[test]
+#[ignore = "auth() needs to be worked on"]
 fn coblox_bitcoincore_getnewaddress() {
     let _ = pretty_env_logger::try_init();
     let docker = clients::Cli::default();
-    let node = docker.run(images::coblox_bitcoincore::BitcoinCore::default());
+    let node = docker.run(RunnableImage::from(
+        images::coblox_bitcoincore::BitcoinCore::default(),
+    ));
```

Comment:
Can we define the `run` API in such a way that we don't need to type `RunnableImage::from`? Does this compile?

```rust
impl<I: Image> Into<RunnableImage<I>> for I {

}
```

I think that should allow users to pass instances of `Image` into `Docker::run`.

Reply:
Sorry for the confusion. I wasn't paying attention to details while I was fixing the tests. The `run` API already gets `image` param as `impl Into<RunnableImage<I>>`. So, just `docker.run(images::coblox_bitcoincore::BitcoinCore::default())` will work since we already have `From` trait implemented for `RunnableImage<I>` to convert from `I: Image` Right?

Reply:
Ah right, of course!

Lets please use that notation then so newcomers learn about this when looking at the tests!